amulety/bcr_embeddings.py

```diff
@@ -22,36 +22,61 @@
 
 
 def antiberty(
-    sequences: pd.Series,
+    sequences,
     cache_dir: Optional[str] = None,
     batch_size: int = 50,
 ):
     """
-    Embeds sequences using the AntiBERTy model.\n
+    Embeds sequences using the AntiBERTy model.
     The maximum length of the sequences to be embedded is 510.
+
+    Parameters:
+        sequences: pd.Series for single chain or pd.DataFrame for H+L mode
     """
     from antiberty import AntiBERTyRunner
 
     max_seq_length = 510
 
-    X = sequences
-    X = X.apply(lambda a: a[:max_seq_length])
+    # Handle both Series (single chain) and DataFrame (H+L) inputs
```

Comment:
Why is this needed? The user already provides the column that should be used for the sequence and we should not update this again here. This function should now just embed all the provided sequences.